ocs_ci/utility/rosa.py

```diff
@@ -462,6 +464,7 @@ def install_odf_addon(cluster):
         raise ManagedServiceAddonDeploymentError(
             f"Addon {addon_name} failed to be installed"
         )
+    post_onboarding_verification()
```

Comment:
This is tested also as part of acceptance. I am not sure if we want to test it twice.